kraken/litmus/common_litmus.py

```diff
@@ -0,0 +1,97 @@
+import kraken.invoke.command as runcommand
+import yaml
+import logging
+import time
+import sys
+
+
+def get_pod_ready_status():
+    pods_status = runcommand.invoke("kubectl get pods -n litmus -o yaml")
```

Comment:
I'd check the readiness of litmus by querying the deployment status since it is deployed as an operator, e.g.:
```
oc wait deploy -n litmus chaos-operator-ce --for=condition=Available --timeout=30s
```